### PR TITLE
feat: parse field plugin text to return its value or empty string

### PR DESCRIPTION
**Pull request by molebox**
Im not sure what prefix to add to this? feat? fix? Also wasnt sure if i should add tests?

### CODE REVIEW

**Comment by ncphillips** [comment]
I would probably go with the `fix` prefix.

Don't worry about writing automated tests. This would be a particularly odd API to write automated tests for. It's something we could look into, but I wouldn't ask that for this PR.

File: packages/tinacms/src/plugins/fields/ColorFieldPlugin.tsx [hunk 1]
```diff
@@ -42,4 +43,5 @@ export const ColorField = wrapFieldsWithMeta<InputProps, ColorFieldProps>(
 export default {
   name: 'color',
   Component: ColorField,
+  parseString,
```
Comment (ncphillips) [inline packages/tinacms/src/plugins/fields/ColorFieldPlugin.tsx]: Looks good! The name of the property being exported should be `parse` though.

```suggestion
  parse: parseString
```

Comment (molebox) [reply packages/tinacms/src/plugins/fields/ColorFieldPlugin.tsx]: Ah ok, cool!

**Comment by molebox** [comment]
> I would probably go with the `fix` prefix.
> 
> Don't worry about writing automated tests. This would be a particularly odd API to write automated tests for. It's something we could look into, but I wouldn't ask that for this PR.
> 
> You can test it manually by spinning up [`demo-gatsby`](https://github.com/tinacms/tinacms#development) and clearing values in the [author](https://github.com/tinacms/tinacms/blob/master/packages/demo-gatsby/src/components/bio.js) or [blog post](https://github.com/tinacms/tinacms/blob/master/packages/demo-gatsby/src/templates/blog-post.js) forms..

ok nice, didnt see you added this check. I'll do a manual test

**Comment by molebox** [comment]
Im getting: Error: Unable to find plugin "gatsby-tinacms-teams". Perhaps you need to install its package? When i try and run the demo-gatsby... :-(

**Comment by ncphillips** [comment]
Hmm, just to make sure. Did you run `npm run bootstrap` and `npm run build` from the root of the repository before starting `demo-gatsby`.

**Comment by ncphillips** [comment]
We wound up with duplicate PRs. @NickCis also opened #638.

They're effectively the same so I'll make sure yah's get both get cred

**Comment by NickCis** [comment]
@ncphillips Ohh, sorry, my bad, as #632 didn't have a link to a PR I didn't see it :disappointed:

**Comment by molebox** [comment]
I didn't run npm run build. I will give that a shot when I get to my computer.

Re duplicate PR: ahh should I have linked this PR to the issue as fixed? My bad, would have been cleaerer.

**Comment by molebox** [comment]
> Hmm, just to make sure. Did you run `npm run bootstrap` and `npm run build` from the root of the repository before starting `demo-gatsby`.

That was the issue, i didnt run `npm run build` ! 

Seems to work fine, it doesnt break, i can clear all text input forms. Was there a specific error that should occur? Otherwise i think this is good to go :)

File: packages/tinacms/src/plugins/fields/ColorFieldPlugin.tsx (hunk 1 above)
Comment (molebox) [reply packages/tinacms/src/plugins/fields/ColorFieldPlugin.tsx]: These changes have been made in the latest commit

**Comment by ncphillips** [comment]
Without the fix, clearing text will often remove the field from the markdown file

**Comment by molebox** [comment]
> Without the fix, clearing text will often remove the field from the markdown file

So when i remove all the text from the form field in the editor, it should just be blank shouldnt it? Then i can type text again if i wish..? If so, thats what it does now. If this didnt work, when i remove the text from the form field then you are saying the whole field would disappear?

Sorry, just trying to make sure i understand what you mean, and i have a feeling that i dont..

**Comment by ncphillips** [comment]
@all-contributors please add @molebox for code

**Comment by molebox** [comment]
@ncphillips So i guess this PR can be closed now as the other one has been merged... Thanks for adding me to the contributers. I would like to pick up another issue and get some code merged next time! :)

**Comment by ncphillips** [comment]
Your welcome! Thanks for putting the effort in 😄 I'm looking forward to your next PR!

**Pull request closed without merging**